src/size.rs

```diff
@@ -444,6 +433,22 @@ impl<T, U> Into<mint::Vector2<T>> for Size2D<T, U> {
     }
 }
 
+impl<T, U> From<Vector2D<T, U>> for Size2D<T, U> {
+    fn from(v: Vector2D<T, U>) -> Self {
+        Size2D {
+            width: v.x,
+            height: v.y,
+            _unit: PhantomData,
+        }
+    }
+}
+
+impl<T: Copy, U> Into<Vector2D<T, U>> for Size2D<T, U> {
```

Comment:
why are we not implementing `From<Size2D> for Vector2D` instead?

Reply:
I got mixed up about which provide the other, I'll change it.

Reply:
Done.